### PR TITLE
Closes #551: Reimplement JotForms subscribe form in footer

### LINKED ISSUE #551: wire up email subscription form
Re-implement JotForms email subscription form.

Note this will require using `jquery.noConflict()`, as JotForms conflicts with jQuery.

### PR DESCRIPTION
**Pull request by CloudNiner**
The First/Last name fields were left in, but hidden, to remain
compatible with existing form.

Added email validation on keyup so that the subscribe button
changes state while typing. It felt weird to have to click
out of the input box to enable the subscribe button.

Seemingly no issues with re-enabling jQuery.noConflict().

Added some basic styling for subscribe button disabled state.

## Demo

cac-jotform-1
cac-jotform-2

### CODE REVIEW

File: python/cac_tripplanner/templates/home.html [hunk 1]
```diff
@@ -230,7 +230,7 @@
 
 {%block jspage %}
 <script type="text/javascript">
-//jQuery.noConflict(); // because JotForms.
+jQuery.noConflict(); // because JotForms.
```
Comment (KlaasH) [inline python/cac_tripplanner/templates/home.html]: This takes care of the home view, but the footer is used on other views (i.e. Learn list and detail), so they're getting console errors "prototype.forms.js:591 Uncaught TypeError: element.dispatchEvent is not a function" and "prototype.forms.js:575 Uncaught TypeError: element.attachEvent is not a function".

**Comment by CloudNiner** [comment]
RE: the flashing subscribe button. I went for the debounce, as it should generally solve the issue and keeps us from eliminating technically correct addresses.

PR updated.

**Comment by KlaasH** [comment]
Looks good, though this project has lodash, so the hand-crafted debounce isn't strictly necessary.

**Comment by CloudNiner** [comment]
Whoops. Updated to `_.debounce`.

**Pull request merged**